plugins/database/src/db_plugin.cpp

```diff
@@ -8343,27 +8367,39 @@ irods::error db_mod_resc_data_paths_op(
         if ( logSQL != 0 ) {
             log_sql::debug("chlModRescDataPaths SQL 2");
         }
+
+        const auto [secs, millis] = get_current_time();
+
         cllBindVars[cllBindVarCount++] = _old_path;
         cllBindVars[cllBindVarCount++] = _new_path;
+        cllBindVars[cllBindVarCount++] = secs.c_str();
+        cllBindVars[cllBindVarCount++] = millis.c_str();
         cllBindVars[cllBindVarCount++] = rescId;
         cllBindVars[cllBindVarCount++] = oldPath2;
         cllBindVars[cllBindVarCount++] = userName2;
         cllBindVars[cllBindVarCount++] = zoneToUse;
-        status =  cmlExecuteNoAnswerSql(
-                      "update R_DATA_MAIN set data_path = replace (R_DATA_MAIN.data_path, ?, ?) where resc_id=? and data_path like ? and data_owner_name=? and data_owner_zone=?",
-                      &icss );
+        status = cmlExecuteNoAnswerSql(
+            "update R_DATA_MAIN set data_path = replace (R_DATA_MAIN.data_path, ?, ?), modify_ts=?, modify_ts_millis=? "
+            "where resc_id=? and data_path like ? and data_owner_name=? and data_owner_zone=?",
+            &icss);
     }
     else {
         if ( logSQL != 0 ) {
             log_sql::debug("chlModRescDataPaths SQL 3");
         }
+
+        const auto [secs, millis] = get_current_time();
+
         cllBindVars[cllBindVarCount++] = _old_path;
         cllBindVars[cllBindVarCount++] = _new_path;
+        cllBindVars[cllBindVarCount++] = secs.c_str();
+        cllBindVars[cllBindVarCount++] = millis.c_str();
         cllBindVars[cllBindVarCount++] = rescId;
         cllBindVars[cllBindVarCount++] = oldPath2;
-        status =  cmlExecuteNoAnswerSql(
-                      "update R_DATA_MAIN set data_path = replace (R_DATA_MAIN.data_path, ?, ?) where resc_id=? and data_path like ?",
-                      &icss );
+        status = cmlExecuteNoAnswerSql(
+            "update R_DATA_MAIN set data_path = replace (R_DATA_MAIN.data_path, ?, ?), modify_ts=?, modify_ts_millis=? "
+            "where resc_id=? and data_path like ?",
+            &icss);
```

Comment:
Revert these changes.

This code has nothing to do with modifying resource info.

Reply:
Done.